README.md

```diff
@@ -138,6 +163,9 @@ Steps:
 
 You can find detailed steps at the tutorial [Go for beginners: Getting started](https://github.com/Jigsaw-Code/outline-sdk/discussions/67).
 
+## Wrap your website in a SDK-enabled App
```

Comment:
I'm not sure whether we should put this in a dedicated section or within "Add the SDK to Your App". I'll let @fortuna make the call.